notebooks/Overlay_Management_Demo.ipynb

```diff
@@ -28,6 +28,18 @@
     "mast_aladin._overlays_dict.keys()"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c0c0eef0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# we can confirm that these overlays are tracked in the ipyaladin `.overlays` trait, which\n",
+    "# lists the current overlays on the viewer\n",
```

Comment:
I think it would be worth mentioning the semi-delayed nature of updates.  Something like:

```suggestion
    "# lists the current overlays on the viewer\n",
    "# Note: If a cell adds/removes an overlay, this value won't be updated until the cell completes.\n",
```